dev-docs/source/VSCode.rst

```diff
@@ -107,8 +107,9 @@ switched out with the command and various args for whatever you have generated w
             {
                 "label": "Build Mantid",
                 "type": "shell",
-                "command": "ninja",
+                "command": "cmake",
                 "args": [
+                    "--build",
```

Comment:
You do not need the `-C` here. The proper syntax is `cmake --build [path to build]`.